critiquebrainz/frontend/templates/macros.html

```diff
@@ -93,6 +93,19 @@
   </div>
 {%- endmacro %}
 
+{% macro show_tags(tags) %}
+  {% if tags %}
+    {% for tag in tags %}
+      <a href="{{"https://musicbrainz.org/tag/%s"|format(tag['name'])}}">{{tag['name']}}</a>
+      {%- if not loop.last -%}
+        ,
+      {% endif %}
+    {% endfor %}
+  {% else %}
+    <p>{{ _('(none)') }}</p>
```

Comment:
You can also add a `class="text-muted"` to this one to make it look a bit better.